### PR TITLE
IexFinder: Calculation of Extreme Values

### PR DESCRIPTION
**Pull request by zolabar**
IexFinder: Interactive Symbolic Calculation of Extreme Values

[IexFinder](https://mybinder.org/v2/gh/zolabar/IexFinder/main?urlpath=voila%2Frender%2F/IexFinder_voila.ipynb)

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks!

File: _data/gallery.yaml [hunk 1]
```diff
@@ -123,3 +123,10 @@
   ref: 91b8046dfc51e0623700ece1372e64dc770ae71e
   repo_url: https://github.com/sunnysinghnitb/covid-19-world-dashboard
   image_url: https://raw.githubusercontent.com/sunnysinghnitb/covid-19-world-dashboard/master/thumbnail.png
+  
+  - title: IexFinder: Interactive Symbolic Calculation of Extreme Values
```
Comment (jtpio) [inline _data/gallery.yaml]: Looks like we can remove the small indent:

```suggestion
- title: IexFinder: Interactive Symbolic Calculation of Extreme Values
```

File: _data/gallery.yaml [hunk 2]
```diff
@@ -123,3 +123,10 @@
...
+  description: Interactive Symbolic Calculation of Extreme Values
+  url: voila/render/IexFinder_voila.ipynb
+  ref: 32a76389ef77462d92fb0cbe06aea5f004ea64eb
+  repo_url: https://github.com/zolabar/IexFinder
+  image_url: https://github.com/zolabar/IexFinder/blob/main/Figures/exFinder_usage_2.PNG
```
Comment (jtpio) [inline _data/gallery.yaml]: We'll need the link to the raw image here:

```suggestion
  image_url: https://github.com/zolabar/IexFinder/raw/main/Figures/exFinder_usage_2.PNG
```

**Comment by zolabar** [comment]
@jtpio, thanks for the revisons, I included them.

**Review by jtpio — APPROVED** [review]
Thanks!!

**Comment by zolabar** [comment]
@jtpio , thank you!

**Pull request merged**